api/src/main/java/com/minirogue/api/media/MediaEnums.kt

```diff
@@ -89,5 +89,11 @@ enum class Company {
     ELECTRONIC_ARTS,
 
     @SerialName("Golden Books")
-    GOLDEN_BOOKS;
+    GOLDEN_BOOKS,
+
+    @SerialName("Random House Audio")
+    RANDOM_HOUSE_AUDIO,
+
+    @SerialName("Random House Worlds")
+    RANDOM_HOUSE_WORLDS;
```

Comment:
These are being added for a database update. Random House Worlds is a new branding for Random House's licensed books. Random House Audio was listed as the publisher of one of the recent audio books, and I'm not sure if this is new/different from regular "Random House", but I'm adding it now and can roll back usages of it easily enough later if need be.